lib/el_golang_lib/hook/fee_hook.go

```diff
@@ -188,3 +193,27 @@ func zeroesAndOnes(data []byte) (int64, int64) {
 	}
 	return zeroes, ones
 }
+
+func getStorageSlots() (common.Hash, common.Hash, common.Hash) {
+	layout, err := bindings.GetStorageLayout("L1Oracle")
+	if err != nil {
+		panic("could not get storage layout for L1Oracle")
+	}
+
+	baseFeeEntry, err := layout.GetStorageLayoutEntry("baseFee")
+	if err != nil {
+		panic("could not get basefee storage slot")
+	}
+	overheadEntry, err := layout.GetStorageLayoutEntry("l1FeeOverhead")
+	if err != nil {
+		panic("could not get overhead storage slot")
+	}
+	scalarEntry, err := layout.GetStorageLayoutEntry("l1FeeScalar")
+	if err != nil {
+		panic("could not get scalar storage slot")
+	}
+
+	return common.BigToHash(new(big.Int).SetUint64(uint64(baseFeeEntry.Slot))),
```

Comment:
one thing im not sure is that why https://github.com/SpecularL2/specular/blob/dc1fac0b07720b8fdbe332a66369074503f86327/ops/bindings/solc/types.go#L81 (comes from optimism). should we change it to string and parse to common.Hex?